### PR TITLE
Crud_api

### PR DESCRIPTION
**Pull request by amanpreetchalga**
* trying without mongoose (following tutorial)
* able to connect and perform crud operations on the db present on atlas
* errors connecting to db stating db.collection is not a function

### CODE REVIEW

**Review by amanpreetchalga — COMMENTED** [review]
* added few packages in package.json 
* added db.js and changed server.js accordingly

**Review by amanpreetchalga — COMMENTED** [review]
* added packages
* added db.js and changed server.js accordingly

File: apps/backend/src/connect.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+const { MongoClient } = require("mongodb");
+// Replace the following with your Atlas connection string                                                                                                                                        
+const url = "mongodb+srv://helping_hand:X26igJL9@clusterhelpinghand.c1myh2s.mongodb.net/?retryWrites=true&w=majority";
```
Comment (Jordan-Patel) [inline apps/backend/src/connect.js]: I think you aren't using this file from what I can see, if that is the case, maybe delete the file. If it is needed, then please remove the db url

**Comment by amanpreetchalga** [comment]
closing pull request with required changes

**Pull request merged**